### PR TITLE
Fix params passing

### PR DESCRIPTION
**Pull request by ElArkk**
The passing of params in evotuning was handled inconsistently.
This PR changes this, such that `params` now always expects the full tuple
of weights: (mlstm1900, (), Dense, ()).

Weight dumping got adjusted accordingly, and there is a new convenience function
`load_params`, that by defaults loads all weights from the paper, and can also load previously dumped weights when supplied a folder path.

### CODE REVIEW

File: jax_unirep/utils.py [hunk 1]
```diff
@@ -160,49 +188,50 @@ def get_embeddings(sequences: List[str]) -> np.ndarray:
...
     from the output of the mLSTM1900.
     """
+    if folderpath:
+        weights_1900_dir = Path(folderpath)
+    else:
+        weights_1900_dir = Path(
+            pkg_resources.resource_filename(
+                "jax_unirep", "weights/1900_weights/uniref50"
+            )
+        )
```
Comment (ericmjl) [inline jax_unirep/utils.py:205]: This looks duplicated. Can we have a single source of truth the default weights dir, wrapped in a function?

File: jax_unirep/utils.py [hunk 2]
```diff
@@ -160,49 +188,50 @@ def get_embeddings(sequences: List[str]) -> np.ndarray:
...
     """Load pre-trained mLSTM1900 weights from the UniRep paper."""
+    if folderpath:
+        print(folderpath)
+        weights_1900_dir = Path(folderpath)
+    else:
+        weights_1900_dir = Path(
+            pkg_resources.resource_filename(
+                "jax_unirep", "weights/1900_weights/uniref50"
+            )
+        )
```
Comment (ericmjl) [inline jax_unirep/utils.py:222]: Duplicated (echoing other comment above).

File: jax_unirep/utils.py [hunk 3]
```diff
@@ -235,9 +264,18 @@ def validate_mLSTM1900_params(params: Dict):
...
+        load_params_1900(folderpath=folderpath),
+        (),
+        load_dense_1900(folderpath=folderpath),
+        (),
+    )
+
+
+# def load_embeddings(path: str = None):
+#     """Load embeddings of amino acids for the uniref50 model."""
+#     return np.load(weights_1900_dir / "embed_matrix:0.npy")
```
Comment (ericmjl) [inline jax_unirep/utils.py:278]: Should this be removed?

File: jax_unirep/evotuning.py [hunk 4]
```diff
@@ -168,7 +167,7 @@ def fit(
     step_size: float = 0.001,
     holdout_seqs: Optional[List[str]] = None,
     proj_name: Optional[str] = "temp",
-    steps_per_print: Optional[int] = 200,
+    steps_per_print: Optional[int] = None,
```
[the remark this answers is not in the document]
Comment (ElArkk) [reply jax_unirep/evotuning.py:170]: If we default it to a number, there will be weight dumping happening per default. But I don't have strong opinion on whether this is desirable. So I can change it back.

File: jax_unirep/utils.py (hunk 3 above)
Comment (ElArkk) [reply jax_unirep/utils.py:278]: Yes :)

File: jax_unirep/evotuning.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (ericmjl) [reply jax_unirep/evotuning.py:170]: Ah yeah, I forgot to mention - dumping by default is a pretty sane prior to have. Amongst deep learners, this is a fairly idiomatic practice. Recently saw a talk by Andrej Karpathy where I was reminded about this point: https://slideslive.com/38917690/multitask-learning-in-the-wilderness

File: README.md [hunk 5]
```diff
@@ -91,36 +91,37 @@ Given a set of starter weights for the mLSTM (defaults to
...
-optimal learning rate.
-Example usage:
+There are two functions with differing levels of control available.
 
-```python
-from jax_unirep import evotune
+The [`evotune`][evotunefunc] function uses `optuna` under the hood
+to automatically find: 
+1. the optimal number of epochs to train for, and
+2.  the optimal learning rate,
```
Comment (ericmjl) [inline README.md:99]:
```suggestion
2. the optimal learning rate, 
```

**Comment by ericmjl** [comment]
MERGING!
